specifyweb/frontend/js_src/lib/components/Syncer/syncers.ts

```diff
@@ -658,7 +659,7 @@ export const syncers = {
             ]) as TYPE_MAPPER[keyof TYPE_MAPPER]) ?? ('Unknown' as const);
         const spec = mapper[type] ?? mapper.Unknown;
         const { serializer } = syncers.object(
-          spec(cell, { ...extraPayload, rawType })
+          spec(cell, extraPayload, rawType)
```

Comment:
this was causing `extraPayload` object to loose it's prototype chain, thus table was loosing the `getRelationship` and all other methods